GSA-TTS/FAC
# Enable djLint "Django" Profile

# Enable djLint "Django" Profile

No issue.

djLint is our HTML linter. Previously, it had the default "HTML" profile enabled. By changing the profile to "Django," we get much better coverage for template tags, filters, and Django conventions. 

## Changes:
1. Add `profile="django"` and `use_gitignore=true` to the djLint section of `pyproject.toml`
2. Resolve all the minor errors that pop up due to this change
    * 29 out of 72 linted files had very minor changes
    * 1 file (`sidenav.html`) was pretty much totally restructured

## How to test:
1. Switch to this branch and run normally
2. Check out the app, verify that things look okay and that links are still working
3. Maybe run Cypress as a general verification
    * Disabling `testSubmissionAccess` in `full-submission.js` line 136 will avoid some local funkyness


## PR checklist: submitters


## PR checklist: reviewers


The larger the PR, the stricter we should be about these points.
